src/player.js

```diff
@@ -0,0 +1,10 @@
+const Gameboard = require('./gameboard');
+
+class Player {
+	constructor(type) {
+		this.type = type,
+		this.gameboard = new Gameboard()
```

Comment:
Initializing a class inside another isn't a good OOP practice. Let us just inject an already instantiated `Gameboard` object as a dependency here in `Player`.